src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { useParams } from 'react-router-dom';
+
+const getPeopleWithParents = (people: Person[]) => {
+  return people.map(person => {
+    return {
+      ...person,
+      father: people.find(f => f.name === person.fatherName),
+      mother: people.find(m => m.name === person.motherName),
+    };
+  });
+};
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const { personSlug } = useParams<string>();
+
+  useEffect(() => {
+    setLoading(true);
+    setError(false);
+
+    getPeople()
+      .then(p => setPeople(getPeopleWithParents(p)))
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {loading ? (
+            <Loader />
+          ) : !!people.length ? (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+              <tbody>
+                {people.map(person => {
+                  return (
+                    <tr
+                      key={person.slug}
+                      className={
+                        personSlug === person.slug
+                          ? 'has-background-warning'
+                          : ''
+                      }
```

Comment:
it's better to use classnames in cases like that, it allows you to dynamically toggle classes based on state or props without manually concatenating strings, making the code more maintainable and easier to read.